test/gtest/common/test_helpers.cc

```diff
@@ -469,6 +469,26 @@ bool is_interface_usable(struct ifaddrs *ifa)
            !netif_has_sysfs_file(ifa->ifa_name, "wireless");
 }
 
+
+int get_proc_self_status_field(const std::string &parameter)
```

Comment:
size_t / ssize_t

Reply:
I use `int` to return `-1` error code.

Reply:
btw can use `ssize_t` as Yossi commented